src/shared/enums/role.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum Role {
+  SITE_ADMIN = 'site-admin',
```

Comment:
That was/will be changed to `admin`

Reply:
Ok, when?

Reply:
yesterday or so

Reply:
https://github.com/MiSArch/keycloak/pull/9

Reply:
```suggestion
  SITE_ADMIN = 'admin',
```